backend/trafficrules/src/main/java/com/coffeejug/trafficrules/service/StatisticService.java

```diff
@@ -0,0 +1,13 @@
+package com.coffeejug.trafficrules.service;
+
+import org.springframework.stereotype.Service;
+
+import java.lang.management.ManagementFactory;
+
+@Service
+public class StatisticService {
+
+    public long getUptime() {
+        return ManagementFactory.getRuntimeMXBean().getUptime();

```

Comment:
It's better to have this injected as a field and have a spring bean for it. Getting dependencies fetched from the static methods makes code less testable and more coupled